### DISCUSSION TITLE
Add standard OMSA binary paths to drac* modules

### DISCUSSION THREAD

**Discussion by clallen** [saltstack/salt#67646] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
The standard install path for Dell's OpenManage Server Administrator (OMSA) is /opt/dell/srvadmin.
Since this path is rarely ever in a system-wide path, the drac modules often fail with "racadm binary not in path".

**Describe the solution you'd like**
I would like to add the srvadmin bin paths to `__virtual__()`, to give the modules a better chance of finding the binaries.

**Describe alternatives you've considered**
I thought about using environ.setval to update the path before calling the drac functions, but that involves setting up something outside the modules that directly affects their ability to load.
Also, since this is specific to the drac modules functionality it makes more sense to me to have them know about the common path where the binaries are installed.

**Additional context**
I have a fork with commits ready for a pull request if this is something the Salt team thinks will be useful.

**Comment by garethgreenaway** [comment]
@clallen Thanks for the report.  We would definitely welcome a PR that provides a fix.  I wonder if a better approach for a fix might be to check first to see if the binaries are available in the path and if they are not to use the default location of `/opt/dell/srvadmin`.  It could even be a configurable option in the minion configuration.
